pkg/metastore/kv.go

```diff
@@ -85,10 +85,18 @@ func MakeLocationID(l *pb.Location) string {
 		// 0. This works out well as the key is byte aligned to the nearest 8
 		// bytes that way.
 
-		//nolint:errcheck // ignore error as writing to the hash will cannot error
+		//nolint:errcheck,staticcheck
+		// ignore the error as writing to the hash will not error
+		// https://staticcheck.io/docs/checks#SA1003
+		// The encoding/binary package can only serialize types with known sizes.
+		// This precludes the use of the int and uint types, as their sizes differ on different architectures.
+		// TODO: Fix this.
```

Comment:
I don't understand what needs to be fixed here.`hash` is fixed size, so the staticcheck link doesn't apply here.

Reply:
If I don't understand this incorrectly, `int` is arch dependent so var sized and staticcheck wants us to specify it.

Reply:
Ahhhhhh I see now. Then we need to change this to `uint64(0)` and `uint64(1)`.